Web Inspector: Update CSSDocumentation.js with upstream data. June 2022

rcaliman-apple opened the pull request:
#### 87f52cfe8ff62a6fa83c4b985dd72c9c175783c7
<pre>
Web Inspector: Update CSSDocumentation.js with upstream data. June 2022
<a href="https://bugs.webkit.org/show_bug.cgi?id=242026">https://bugs.webkit.org/show_bug.cgi?id=242026</a>

Reviewed by Patrick Angle.

Sync `CSSDocumentation.js` with latest data from upstream source: <a href="https://github.com/microsoft/vscode-custom-data">https://github.com/microsoft/vscode-custom-data</a>

* Source/WebInspectorUI/UserInterface/External/CSSDocumentation/CSSDocumentation.js:

Canonical link: <a href="https://commits.webkit.org/251875@main">https://commits.webkit.org/251875@main</a>
</pre>

patrickangle reviewed (approved):
Did we also confirm that all of our existing overrides are still necessary with the latest data?

rcaliman-apple committed:
Web Inspector: Update CSSDocumentation.js with upstream data. June 2022
https://bugs.webkit.org/show_bug.cgi?id=242026

Reviewed by Patrick Angle.

Sync `CSSDocumentation.js` with latest data from upstream source: https://github.com/microsoft/vscode-custom-data

* Source/WebInspectorUI/UserInterface/External/CSSDocumentation/CSSDocumentation.js:

Canonical link: https://commits.webkit.org/251875@main
